sdk/turing/generated/model/traffic_rule_condition.py

```diff
@@ -115,7 +115,7 @@ def __init__(self, field_source, field, values, *args, **kwargs):  # noqa: E501
 
         Args:
             field_source (FieldSource):
-            field (str): If `field_source` is `header`, then `field` should contain the name of the request header, otherwise, if `field_source` is `payload`, then `field` should be a valid json path 
+            field (str): For HTTP_JSON protocol, the valid `field_source` are `header` and `payload`. Whereas, for UPIV1 protocol the valid `field_source` are `header` and `prediction_context` If `field_source` is `header`, then `field` should contain the name of the request header, otherwise, if `field_source` is `payload`, then `field` should be a valid json path. If `field_source` is `header`, then `field` should contain the name of the request header, otherwise, if `field_source` is `prediction_context`, then `field` should contains variable name. 
```

Comment:
> If `field_source` is `header`, then `field` should contain the name of the request header

I think this part of the comment is repeated. From the way the description is written [here](https://github.com/caraml-dev/turing/blob/grpc_upi/api/api/specs/routers.yaml#L723), it's not clear that the lines 2 and 3 correspond to different protocols and I'm not sure it's important to distinguish. Shall we simply remove the duplicated line?

>           description: >
            For HTTP_JSON protocol, the valid `field_source` are `header` and `payload`. Whereas, for UPIV1 protocol the valid `field_source` are `header` and `prediction_context`
            If `field_source` is `header`, then `field` should contain the name of the request header. If `field_source` is `payload`, then `field` should be a valid json path. If `field_source` is `prediction_context`, then `field` should contains variable name.

Reply:
Yeah, rephrased the documentation.